### PR TITLE
Multi namespace

### PR DESCRIPTION
**Pull request by qbalukom**
- add "all namespaces" option  in namespace dropdown
- display resources from all namespaces when in "all namespaces" mode
- show namespace resources from all namespaces in command palette
- show "all namespaces" option when typing "ns all", "ns -a" or "ns *" in command palette

### CODE REVIEW

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
As we have a talk, we need to catch something like this error 
image

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
typing `cmk +k` and `ns all` navigates all the time to namespace overview, when I am on for example Pods view and type in command pallet `ns all` I need to have two options: switch (redirect to pods view for all namespaces) and navigate (redirect to namespace overview)

**Review by mrCherry97 — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

File: public/i18n/en.yaml [hunk 1]
```diff
@@ -240,6 +240,7 @@ command-palette:
     open-preferences: Open Preferences
     resource-aliases: Built-in Resource Aliases
     short-help: Type to search, e.g. "ns default" or "pod nginx"
+    all-namespaces: List resources from all namespaces
```
Comment (NHingerl) [inline public/i18n/en.yaml:243]:
```suggestion
    all-namespaces: List resources from all Namespaces
```

Ignore this if we don't capitalize "Namespace" in Busola.

**Review by mrCherry97 — DISMISSED** [review]

**Review by NHingerl — DISMISSED** [review]

**Review by mrCherry97 — DISMISSED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**